clients/cobol-lsp-vscode-extension/schema/proc_grps.schema.json

```diff
@@ -29,7 +29,80 @@
             "type": "array",
             "description": "List of folders that contain copybooks.",
             "items": {
-              "type": "string"
+              "anyOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "object",
+                  "properties": {
+                    "dataset": {
+                      "type": "string",
+                      "description": "Name of a data set located on the mainframe."
+                    },
+                    "profile": {
+                      "type": "string",
+                      "description": "Profile name to be used by the Zowe Explorer extension"
+                    }
+                  },
+                  "required": ["dataset"]
+                },
+                {
+                  "type": "object",
+                  "properties": {
+                    "ussFile": {
+                      "type": "string",
+                      "description": "USS directory/file on the mainframe."
```

Comment:
Is specifying a file an actual use case?

Reply:
Yes, it is in requirements

Reply:
Yes, the requirement is to make USS files available - by specifying a directory they are in.